Lib/turtledemo/clock.py

```diff
@@ -52,62 +51,54 @@ def clockface(radius):
             jump(-radius)
         rt(6)
 
+def display_date_time():
+    writer.clear()
+    now = datetime.now()
+    writer.home()
+    writer.forward(distance=65)
+    writer.write(now.strftime(format="%A"),
+                 align="center", font=("TkFixedFont", 14, "bold"))
```

Comment:
After adding "dtfont = "TkFixedFont", 14, "bold" above (perhaps after d-t import),

```suggestion
    writer.write(now.strftime(format="%A"), align="center", font=dtfont)
```
and ditto for next write.